spec/lib/api/actions_builder_spec.rb

```diff
@@ -0,0 +1,118 @@
+RSpec.describe Api::ActionsBuilder do
+  let(:role) { FactoryGirl.create(:miq_user_role) }
+  let(:group) { FactoryGirl.create(:miq_group, :miq_user_role => role) }
+  let(:user) { FactoryGirl.create(:user, :miq_groups => [group]) }
+  let(:request) { instance_double(Api::BaseController::Parser::RequestAdapter) }
+
+  before do
+    User.current_user = user
+  end
+
+  describe '#collection_actions' do
```

Comment:
It looks like the only public method is `#actions` - perhaps these example groups need to be grouped under a single `#actions` group?